recipes-apps/ovmenu-ng-skripts/files/delete-igc.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+USB_PATH="/usb/usbstick/openvario/igc"
+IGC_PATH="/home/root/.xcsoar/logs"
+
```

Comment:
I may not have whole context.
Why not "rm -fv $IGC_PATH/*.igc"?

Reply:
Hi Caz, we decided to remove deletion alltogether.